docs: restore to new project docs (#30525)
supabase/supabase: apps/www/_blog/2024-12-06-restore-to-a-new-project.mdx

---
title: 'Restore to a New Project'
description: Effortlessly Clone Your Supabase Project
author: crispy1975
image: launch-week-13/

> thumb: launch-week-13/

Author: Need to add proper assets here.

---
title: 'Restore to a New Project'

> description: Effortlessly Clone Your Supabase Project

Reviewer: ```suggestion
description: Effortlessly Clone Data into Another Supabase Project
```

Author: Adjusted to `Effortlessly Clone Data into a New Supabase Project` as I don't want to indicate this can be any project, but a _new_ project.

author: crispy1975
image: launch-week-13/
thumb: launch-week-13/
categories:
  - developers
  - postgres
tags:
  - cloning
  - postgres
date: '2024-12-06T00:00:01'
toc_depth: 3
launchweek: '13'
---

> Today we’re adding [Restore to a New Project](https://supabase.com/docs/guides/platform/backups#restore-to-a-new-project) as a new tool to make life easier for those who need to copy data easily from an existing Supabase project to a new one. Restore to a New Project integrates seamlessly with daily physical backups and Point-in-Time Recovery (PITR) to provide flexible restoration options where historically there have been some pain points. It is now possible to copy data to a brand new Supabase project in a few clicks.

Reviewer 2: ```suggestion
Today we’re adding [Restore to a New Project](https://supabase.com/docs/guides/platform/backups#restore-to-a-new-project).

You can use this new tool to copy data easily from an existing Supabase project to a new one. **Restore to a New Project** integrates seamlessly with daily physical backups and Point-in-Time Recovery (PITR) to provide flexible restoration options.
```

Author: Done.

<Admonition>

> Restore to a New Project requires physical backups to be enabled, and is not available to customers who are currently on the free tier.

Reviewer: ```suggestion
Restore to a New Project requires physical backups to be enabled, and is only available to customers who are on a paid plan.
```

Author: Done.

> Restore to a New Project requires physical backups to be enabled, and is only available to customers who are on a paid plan.

Reviewer 2: ```suggestion
Restore to a New Project requires physical backups to be enabled. It is available to all customers on a paid plan.
```

Author: Done.

</Admonition>

## How it Works

> When physical backups are enabled for a project Supabase triggers daily backups of project data. Using these snapshots a specific backup can be selected and restored to a new project that is provisioned specifically for the task. This new project will match the following source project attributes.

Reviewer 2: ```suggestion
When physical backups are enabled, Supabase triggers daily backups of project data. You can use this backup to restore to a new Supabase project. The new project should match the original project attributes:
```

- Size of compute instance.
- Disk settings; volume type, size, IOPS, and throughput.
- SSL enforcement configuration.
- Network restrictions.
- Cloud region.

> Once the desired backup is chosen, along with a new project name and secure password, the whole process is automated. The length of time for a new project to fully provision will depend on the size of the source dataset, for smaller projects the process is very quick and the new project will be available in as little as 10 minutes. For larger datasets that run into the hundreds of gigabytes, or terabytes, the process will take longer.

Reviewer 2: ```suggestion
After launching your restored project, the rest of the process is automated. The length of time for a new project to provision will depend on the size of the source dataset. 
```

Author: Done.

> The new project will be made available in the Supabase dashboard as soon as the copy process has completed. From this point the new project will behave exactly as any other Supabase project and is completely independent of the source.

Reviewer: ```suggestion
The new project will be made available in the Supabase Dashboard as soon as the copy process has completed. From this point the new project will behave exactly as any other Supabase project and is completely independent of the source.
```

Author: Done.

> The new project will be made available in the Supabase Dashboard as soon as the copy process has completed. From this point the new project will behave exactly as any other Supabase project and is completely independent of the source.

Reviewer 2: ```suggestion
The new project will be available in the Supabase Dashboard as soon as the copy process has completed. The new project will behave as any other Supabase project and is completely independent of the source.
```

Author: Done.

### Point-in-Time Recovery

In addition to daily backups it is possible to restore from a project with PITR enabled. This allows for very fine granularity when selecting the desired point to restore from. The process is very similar as with daily backup with the exception of being asked to select a specific time.

## Unlimited Cloning from a Source Project

To ensure maximum flexibility a source project can be _copied_ as many times as required, making the tool perfect for testing, development environments etc. However, please note that cloning from an already cloned project is not currently supported (this is in the works).

<Admonition>

New projects created with this process, as with any new Supabase project, will incur the usual compute and disk costs. These costs are displayed ahead of time to ensure there are no surprises.

</Admonition>

## Accessing Restore to a New Project

> The Restore to a New Project feature can be found on the Supabase dashboard under [database backups](https://supabase.com/project/_/database/backups/restore-to-new-project).

Reviewer: ```suggestion
The Restore to a New Project feature can be found on the Supabase dashboard under [database backups](https://supabase.com/dashboard/project/_/database/backups/restore-to-new-project).
```

Author: Done.

<Admonition>

> Please be aware that Restore to a New Project is currently in public alpha and any issues encountered should be reported to Supabase support.

Reviewer: ```suggestion
Please be aware that Restore to a New Project is currently in Private Alpha. You can reach out to [Supabase support](https://supabase.help) to get early access and report any issues.
```

Author: Done.

---
title: 'Restore to a New Project'
description: Effortlessly Clone Your Supabase Project
author: crispy1975
image: launch-week-13/
thumb: launch-week-13/
categories:
  - developers
  - postgres
tags:
  - cloning
  - postgres
date: '2024-12-06T00:00:01'
toc_depth: 3
launchweek: '13'
---

Today we’re adding [Restore to a New Project](https://supabase.com/docs/guides/platform/backups#restore-to-a-new-project) as a new tool to make life easier for Supabase users who need to migrate data easily to a new Supabase project. Integrating seamlessly with daily physical backups and Point-in-Time Recovery (PITR) to provide flexible restoration options where historically there have been too many pain points. With Restore to a New Project it is now possible to migrate data to a brand new Supabase project in a few clicks.

<Admonition>

Restore to a New Project requires physical backups to be enabled, and is not available to customers who are currently on the free tier.

</Admonition>

## How it Works

> When physcial backups are enabled for a project Supabase triggers daily backups of project data. Using these snapshots a specific backup can be selected and restored to a brand-new project that is provisioned specifically for the task. This new project will match the following source project attributes.

Author: Fixed.
